README.md

```diff
@@ -40,30 +44,34 @@ to replace (rare in English). The project is called "Jujutsu" because it matches
 If you have any questions, please join us on Discord
 [![Discord](https://img.shields.io/discord/968932220549103686.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/dkmfj3aGQN)
 or start a [GitHub Discussion](https://github.com/martinvonz/jj/discussions).
-The [glossary](docs/glossary.md) may also be helpful.
+The [glossary](https://martinvonz.github.io/jj/latest/glossary) may also be helpful.
 
 
 ## Getting started
 
-Follow the [installation instructions](docs/install-and-setup.md) to obtain and configure `jj`.
+Follow the [installation
+instructions](https://martinvonz.github.io/jj/latest/install-and-setup) to
+obtain and configure `jj`.
 
-The best way to get started is probably to go through
-[the tutorial](docs/tutorial.md). Also see the
-[Git comparison](docs/git-comparison.md), which includes a table of
-`jj` vs. `git` commands.
+The best way to get started is probably to go through [the
+tutorial](https://martinvonz.github.io/jj/latest/tutorial). Also see the [Git
+comparison](https://martinvonz.github.io/jj/latest/git-comparison), which
+includes a table of `jj` vs. `git` commands.
 
-As you become more familiar with Jujutsu, the [FAQ](docs/FAQ.md) may help.
+As you become more familiar with Jujutsu, the
+[FAQ](https://martinvonz.github.io/jj/latest/FAQ) may help.
 
 
 ## Features
```

Comment:
Out of scope for this PR: What do you think about moving this into `docs/`?

Reply:
You mean all the features? I think it's nice to have this prominently in the README, essentially as an ad for when a user feels like scrolling down. Without it, the README would be very short, which is not too bad in itself, but it seems worse than the current way we use the space.

Reply:
I was thinking of maybe moving the `demos/` dir under `docs/`, but I am not sure about that either.

Reply:
Yes, I meant moving the features out of the README. It's not that I don't want them in the README, it's just that they seem to be missing from the docs site. I noticed that https://github.com/clap-rs/clap has very little information in the README. But I'm not sure at all, and I think it's out of scope for this PR anyway.

Reply:
`clap` is a library, that's a little different.

Why do you want to move it?

--------------------

I do quite like this section and I'd prefer keeping it. One reason to treat it differently is that I appreciate the screenshots as an overview, but I don't think of it as *documentation* -- it's neither a good reference nor a great tutorial.

For example, https://github.com/facebook/sapling does **not** have screenshots in the README, and I miss having them there. I think I'd appreciate it if their README included a screenshot of both the command-line and the interactive smartlog.

It would *also* make sense on the docs website, but I think having it in the README is important and having duplicate content to keep in sync is not pleasant. OTOH, one technical option is to move README to docs/ but leave it symlinked from the root dir, or find another way to get Github to show it on the main repo page; I'm unsure about this.

In any case, we could think of a better way to refer to the README from the docs website. 

----------

Tangentially related: I'm not very happy with the default website page being `installation-and-setup.md` as I keep being sent to it. It'd be nicer if it was a page with links to a bunch of useful pages. But I haven't thought exactly of what to put there.

### Looking at what other projects do

Some Rust tools do this, some don't, but I always appreciate it. Restricting myself just to tools that *also* have a documentation website, some that do are:

https://github.com/Canop/broot
https://github.com/arxanas/git-branchless 

Some that don't are

Sapling
https://github.com/rust-lang/cargo
https://github.com/watchexec/watchexec

Almost every smaller tool (without a docs website) does have screenshots (https://github.com/slowli/term-transcript, https://github.com/sharkdp/bat)

Reply:
> Why do you want to move it?

It's not that I want to move it, it's that I feel like the information is missing in the documentation website. I.e., it feels like https://martinvonz.github.io/jj should have information what jj even is. Maybe that's what the landing page should be. So maybe we can duplicate the information between the documentation site and the README, hopefully without actually duplicating the maintenance work for us.

Reply:
Makes sense. For now, I'm still considering https://github.com/martinvonz/jj/ as jj's "official website" and https://martinvonz.github.io/jj as the "documentation website". If we fix the landing page of https://martinvonz.github.io/jj website sufficiently, we could change this. In the long term, that's probably a good idea.

Reply:
I'll resolve this for now.